src/commonMain/kotlin/wizard/files/composeApp/ModuleBuildGradleKts.kt

```diff
@@ -11,9 +11,17 @@ class ModuleBuildGradleKts(info: ProjectInfo) : ProjectFile {
         val otherDeps = mutableSetOf<Dependency>()
         val commonTestDeps = mutableSetOf<Dependency>()
         val otherTestDeps = mutableSetOf<Dependency>()
+        val kspDeps = mutableSetOf<Dependency>()
+        val kspTestDeps = mutableSetOf<Dependency>()
```

Comment:
only kspDeps is enough

Reply:
do you know test ksp libraries? let's keep it simple for now

Reply:
All right. I will work without test